src/components/PriceInputRow.vue

```diff
@@ -2,11 +2,12 @@
   <v-row>
     <v-col :cols="priceForm.price_is_discounted ? '6' : '12'" sm="6">
       <v-text-field
-        v-model="priceForm.price"
+        :model-value="priceForm.price"
+        @update:model-value="newValue => priceForm.price = fixComma(newValue)"
         :label="priceForm.price_is_discounted ? $t('PriceForm.LabelDiscounted') : $t('PriceForm.Label')"
-        type="number"
+        type="text"
```

Comment:
the smartphone numeric keyboard is still displayed ?

Reply:
maybe add a pattern ? I tested and I could input letters, sure there is the error message but better to restrict in the first place no ?

Reply:
`pattern="[0-9]+([\.,][0-9]+)?"`

Reply:
Yes, the numeric keyboard is still displayed because of `inputmode="decimal"`.

The `pattern` didn't work in a useful way, at least on Firefox. Also, I think it's better to show an error rather than silently drop characters from the user input, e.g. "-5.99 bananas (return)" is good to catch rather than silently convert to "5.99". Same for "26 130" which may be two numbers etc.

I added an error message regarding spaces.